### ISSUE TITLE
Re-evaluate `DATA_UPLOAD_MAX_MEMORY_SIZE`

### ISSUE THREAD

**Issue by jnm** [kobotoolbox/kobocat#618]
Not needed until Django gets upgraded, but I think 10 MB as specified in https://github.com/kobotoolbox/kobocat/pull/615/files is not enough. I think we currently allow submission attachments up to 100 MB, but we should confirm. Am I right that under Django 1.8 this is only governed by the NGINX configuration?

**Comment by noliveleger** [comment]
@jnm I've set 10 MB because of the max allowed in Enketo. AFAIK, you are correct.

**Comment by jnm** [comment]
Right. Enketo does limit to 10 MB but Collect will go higher: https://community.kobotoolbox.org/t/kobo-capacity-maxima-for-media-files/1787/5?u=jnm
